File: src/main/java/za/co/shingirai/quizzapp/persistence/model/Question.java
package za.co.shingirai.quizzapp.persistence.model;

import lombok.Data;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Table;
import java.util.List;

@Data
@Entity
@Table(name = "question")
public class Question extends BaseEntity {

    @Column(name = "text")
    private String text;
    @Column(name = "answers")
    private List<Answer> answers;
    @Column(name = "correct_answer")
    private Answer correctAnswer;

}

File: src/main/kotlin/com/github/sechanakira/quizapp/persistence/question.kt
package com.github.sechanakira.quizapp.persistence

import javax.persistence.Column
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id

class Question(val question: String, val answer: String)

class Option(
        @Id
        @GeneratedValue(strategy = GenerationType.AUTO)
        val id: Int,

        @Column(name = "title")
        val title: String,

        @Column(name = "description")
        val description: String)